src/site/content/en/blog/http-cache-security/index.md

```diff
@@ -0,0 +1,177 @@
+or [high resolution timer](https://developer.chrome.com/blog/cross-origin-isolated-hr-timers/)&mdash;to
+pages with [cross-origin
+isolation](/cross-origin-isolation-guide/). 
+
+Modern web browsers enforce [Cross-Origin Embedder Policy
+(COEP)](https://developer.chrome.com/blog/coep-credentialless-origin-trial/). This ensures cross-origin
+resources are either:
+
+*  Public resources, requested without cookies
+*  Resources explicitly allowed to be shared cross-origin, via CORS or the
+   [CORP](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP))
+   header
+
+This setup doesn't prevent an attacker from exploiting Spectre. However, it
+ensures cross-origin resources are not valuable to the attacker (= public
+resource) or allowed to be shared with the attacker (=
+`[CORP](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)):cross-origin`).
+
+### How does HTTP caching affect Spectre?
+
+If the `Cache-Control` header isn't properly set, an attacker could execute an
+attack. For example:
+
+1.  The credentialed resource is cached.
+1.  The attacker loads a cross-origin isolated page.
+1.  The attacker requests the resource again.
+1.  `COEP:credentialless` is set by the browser, so the resource is fetched
+    without cookies. However, a cache may return the credentialled response 
+    instead.
+1.  The attacker can then read the personalized resource using a Spectre
+    attack.
+
+Although a web browser's HTTP cache doesn't allow this type of attack to
+happen in practice, additional caches exist outside of the browser's immediate
+control. This may lead to this attack's success.
+
+## Common misconceptions about HTTP caches
+
+### 1. Resources are cached by the browser only
+
+There are often multiple layers of cache. Some caches are dedicated to a
+single user, some to multiple users. Some are controlled by the server, some
+by the user, and some by intermediaries.
+
+*  **Browser caches**. These caches are owned by and dedicated to a single
+   user, implemented in their web browser. They improve performance by
+   avoiding fetching the same response multiple times.
+*  **Local proxy**. This might have been installed by the user, but can also
+   be managed by intermediaries: their company, their organization, or their
+   internet provider. Local proxies often cache a single response for multiple
+   users, which constitutes a "public" cache. Local proxies have multiple
+   roles.
+*  **Origin server cache / CDN**. This is controlled by the server. The Origin
+   server cache's goal is to reduce the load on the origin server by caching
+   the same response for multiple users. A CDN's goals are similar, but spread
+   across the globe and assigned to the closest set of users to reduce latency.
+
+<figure data-size="full">
+  {% Img
+     class="screenshot",
+     src="image/VbsHyyQopiec0718rMq2kTE1hke2/2VVXjpJLgTwXiPEFTcAI.jpg",
```

Comment:
Do we have a higher resolution image of this? I'd replace.

Reply:
Here is an svg:
https://drive.google.com/file/d/195uEmhdHpSgZWrZ1gFLfLK_HSQtsXWSp/view?usp=sharing
![HTTP Cache   Security Blog Post (1)](https://user-images.githubusercontent.com/4759106/151937899-3f4d7d37-ce8c-4b3a-8357-fe667685211e.svg)

Reply:
Great thank you -- the SVG was added and uploaded. The preview should update soon.